auth0/src/main/java/com/auth0/android/provider/PKCE.java

```diff
@@ -112,14 +113,29 @@ public static boolean isAvailable() {
         return isAvailable(new AlgorithmHelper());
     }
 
+    /**
+     * Checks if this device is capable of using the PKCE flow when performing calls to the
+     * /authorize endpoint, throwing an exception if it does not. This can be used to send
+     * the exception stack trace back to the error handler.
+     *
+     * @throws {@linkplain RuntimeException} if the device does not support PKCE
+     */
+    public static void throwIfNotAvailable() throws IllegalStateException {
+        checkIfAvailable(new AlgorithmHelper());
+    }
+
     @VisibleForTesting
     static boolean isAvailable(@NonNull AlgorithmHelper algorithmHelper) {
```

Comment:
the class is already package-private. What is the reason this method got split into two private and public methods?